target/linux/generic/config-5.10

```diff
@@ -7184,6 +7184,11 @@ CONFIG_ZONE_DMA=y
 # CONFIG_ZPA2326 is not set
 # CONFIG_ZPOOL is not set
 # CONFIG_ZRAM is not set
+# CONFIG_ZRAM_DEF_COMP_LZ4 is not set
+# CONFIG_ZRAM_DEF_COMP_LZ4HC is not set
+# CONFIG_ZRAM_DEF_COMP_LZO is not set
+# CONFIG_ZRAM_DEF_COMP_LZORLE is not set
+# CONFIG_ZRAM_DEF_COMP_ZSTD is not set
```

Comment:
kernel 5.10 does not have these options, they were added with kernel 5.11.

Reply:
Good, then the Kconfig phase will simply remove them.